examples/model_compress/rank_pruner_torch_vgg16.py

```diff
@@ -0,0 +1,176 @@
+import os
+import math
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
+from torchvision import datasets, transforms
+from nni.compression.torch import RankFilterPruner
+
+criterion = 'GeometricMedian'
+
+
+class vgg(nn.Module):
```

Comment:
It is unnecessary to define vgg architecture by yourself.  Suggest to use pytorch provided vgg model structure to simplify the example.

Reply:
Pytorch model usually towards ImageNet dataset which is extremely large and I'll put all the models used in the paper and example into a model folder which should be cleaner.

Reply:
the provided model structure is the same as you defined, you don't have to download the weights.

Reply:
https://github.com/pytorch/vision/blob/master/torchvision/models/vgg.py
set pretrained = False, it will be the same as your code.

Reply:
Pytorch models are pretrained for ImageNet and our implementations/papers are for CIFAR10, they have different weights and different architecture at the fully-connected layers.

Reply:
typically you can replace the classifier